packages/local_auth/local_auth/lib/local_auth.dart

```diff
@@ -91,6 +91,7 @@ class LocalAuthentication {
   /// authentication (e.g. lack of relevant hardware). This might throw
   /// [PlatformException] with error code [otherOperatingSystem] on the iOS
   /// simulator.
+  @Deprecated('Use `requestAuthentication` instead.')
```

Comment:
Is this a mistake? I don't see such a method.

Reply:
This is indeed a mistake, this is something introduced in #4701, it should not apply here yet. Fixing.